content/tutorials/publish-subscribe.textile

```diff
@@ -79,6 +79,36 @@ blang[ruby].
 
   "See this step in Github":https://github.com/ably/tutorials/commit/0ec2c7e
 
+blang[swift].
+  To start using Ably you first need to install the Abbly pod via CocoaPods. Add this line to your application’s Podfile:
```

Comment:
As mentioned in your PR https://github.com/ably/tutorials/pull/12, I do not think this tutorial unfortunately covers the ground needed for someone to get up and running.  This is intended to be a step-by-step guide for someone to get this tutorial working, yet the code below and steps below don't marry up with the tutorial in that there are no instructions on how to set up Xcode, the project, etc.